### ISSUE TITLE
How construct data with predict RNA secondary structure and how predict specific binding location?

### ISSUE THREAD

**Issue by eternal-bug** [kuixu/PrismNet#4]
Hello kuixu!

Thank you very much for developing this tool. After looking at the paper on the tool, there are two questions for me.
1. **How to construct the input data with RNA secondary structure(predict with RNAfold) rather than Icshape experimental data?** (I read the code in GitHub and found that the training data used `seq`, `icshape Score` and `Label`. I had questions about this `icshape Score`. After reading the Icshape-Pipe paper, I couldn't understand how this number was calculated. Is it equal to the probability of secondary structure formation for each base?)
2. **How do you predict specific RBP binding location for a given long RNA?**(Is the prediction generated by sliding window segments?)

**Comment by kuixu** [comment]
You are exactly right! 1) the `icshape` score represents the probability of the nucleic acid being single-stranded in the secondary structure.. 2) For long RNA, PrismNet predicts binding sites by sliding window. By the way, we provide a web server for RBP binding sites prediction using PrismNet, which could automatically deal with long RNA. Link: [http://prismnetweb.zhanglab.net/](http://prismnetweb.zhanglab.net/).

**Comment by eternal-bug** [comment]
Thank you very much. I have another question. What is the **width and step length of the sliding window**?

**Comment by Majimag111** [comment]
It looks like the online service is not available.

**Comment by eternal-bug** [comment]
According to the article code, 
**sliding window width** is fixed, value is 101. **step** is flexible.
